less/index.less

```diff
@@ -49,11 +49,173 @@ table {
 }
 
 /* Set every element's box-sizing to border-box */
+
+
 * {
     box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+html {
+	font-size:62.5%
+	
+}
+
+body {
+	background: #fdf6e4;
+	
+}
+
+.container {
+	font-size:1.6rem;
 }
 
+.container {
+	width: 880px;
+	max-width: 800px;
+	min-width: 660px;
```

Comment:
you generally won't want a min-width on the whole container if you are aiming for a responsive design that supports mobile